testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/jpa/mockprovider/skipquerydetach/persistence.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~  JBoss, Home of Professional Open Source.
+  ~  Copyright 2019, Red Hat, Inc., and individual contributors
+  ~  as indicated by the @author tags. See the copyright.txt file in the
+  ~  distribution for a full listing of individual contributors.
+  ~
+  ~  This is free software; you can redistribute it and/or modify it
+  ~  under the terms of the GNU Lesser General Public License as
+  ~  published by the Free Software Foundation; either version 2.1 of
+  ~  the License, or (at your option) any later version.
+  ~
+  ~  This software is distributed in the hope that it will be useful,
+  ~  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  ~  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+  ~  Lesser General Public License for more details.
+  ~
+  ~  You should have received a copy of the GNU Lesser General Public
+  ~  License along with this software; if not, write to the Free
+  ~  Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+  ~  02110-1301 USA, or see the FSF site: http://www.fsf.org.
+  -->
+
+<persistence xmlns="http://java.sun.com/xml/ns/persistence" version="2.0">
+    <persistence-unit name="queryresultsaremanaged">
+        <provider>org.jboss.as.test.integration.jpa.mockprovider.skipquerydetach.TestPersistenceProvider</provider>
+        <description>Persistence Unit.</description>
+        <jta-data-source>java:jboss/datasources/ExampleDS</jta-data-source>
+        <properties>
+            <property name="jboss.as.jpa.adapterClass" value="org.jboss.as.test.integration.jpa.mockprovider.skipquerydetach.TestAdapter"/>
+            <!-- set skipquerydetach to false so that query results (in non-jta tx) stay associated with the persistence context. -->
```

Comment:
A typo? set skipquerydetach to `true`

Reply:
good catch @zhantemirov I will correct

Reply:
I pushed an update to correct the typo.  Thanks!